submissions/Mifaresss/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,59 @@
+							<path fill="#212121" d="M0 7h7v-7h-7v7zM1 1h5v5h-5v-5z"></path>
+							<path fill="#212121" d="M9 0v7h7v-7h-7zM15 6h-5v-5h5v5z"></path>
+							<path fill="#212121" d="M0 16h7v-7h-7v7zM1 10h5v5h-5v-5z"></path>
+							<path fill="#212121" d="M9 16h7v-7h-7v7zM10 10h5v5h-5v-5z"></path>
+						</svg>
+					</label>
+					<input type="checkbox" id="popap-icon" class="popup-button">
+					<div class="popup">
+						<ul class="popup__list">
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/01.svg" alt="icon1">Спидометр</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/02.svg" alt="icon2">Будильник</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/03.svg" alt="icon3">Якорь</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/04.svg" alt="icon4">Часы</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/05.svg" alt="icon5">Фиг знает</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/06.svg" alt="icon6">Wi-Fi</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/07.svg" alt="icon7">"Стоп"</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/08.svg" alt="icon8">Энергия</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/09.svg" alt="icon9">Батарея</a></li>
+							<li class="popup__item">
+								<label class="sub-label" for="popup-sub-icon"><span>More</span></label>
+								<input type="checkbox" id="popup-sub-icon" class="popup-sub-button">
+								<ul class="popup__sub-list">
+									<li class="popup__sub-item"><a href="#" class="popup__link"><img class="popup__img" src="./img/10.svg" alt="icon10">Блокнот</a></li>
+									<li class="popup__sub-item"><a href="#" class="popup__link"><img class="popup__img" src="./img/11.svg" alt="icon11">Закладка</a></li>
+									<li class="popup__sub-item"><a href="#" class="popup__link"><img class="popup__img" src="./img/12.svg" alt="icon12">Корзина</a></li>
+									<li class="popup__sub-item"><a href="#" class="popup__link"><img class="popup__img" src="./img/13.svg" alt="icon13">Звезда</a></li>
+									<li class="popup__sub-item"><a href="#" class="popup__link"><img class="popup__img" src="./img/14.svg" alt="icon14">USB</a></li>
+									<li class="popup__sub-item"><a href="#" class="popup__link"><img class="popup__img" src="./img/15.svg" alt="icon15">Клевер</a></li>
+								</ul>
+							</li>
+						</ul>
+					</div>
+				</nav>
+			</div>
+			<div class="wrapper__img">
+				<img class="arrow" src="./img/Arrow.png" alt="Стрелочка">
```

Comment:
On the desktop scrollbar overlaps the arrow icon, it looks not ok
![image](https://user-images.githubusercontent.com/16196199/184928949-dd336fe8-cbe4-4a6e-8ece-a24b3c6f7985.png)